package.json

```diff
@@ -91,7 +91,8 @@
     "postpack": "shx rm -f oclif.manifest.json",
     "prepack": "yarn build && oclif manifest && oclif readme",
     "test": "mocha --forbid-only \"test/**/*.test.ts\"",
-    "version": "oclif readme && git add README.md"
+    "version": "oclif readme && git add README.md",
+    "check": "yarn run lint && yarn run format.check && yarn run type.check"
```

Comment:
tests kept failing in the PR for various reasons here, I prefer them to fail locally so I can quick fix. Running the individual commands was cumbersome, added this to check everything before I submit the PR